uiowa/uiowa
# Views Pager ID Bug - use cache instead of tempstore

Resolves https://github.com/uiowa/uiowa/issues/3661

This solution does the same thing as before but captures it as cache for both anonymous and authenticated.

Cache tables have a max so there is less concern of the bin filling up. https://www.drupal.org/node/2526150

I tried to manually purge old dom_ids from the table, but couldn't come up with a "fingerprint" to tie to the cache too without at least temporarily messing up the ajax pager for an anonymous page load.

I set to permanent because some were expiring and then causing issues for cached ajax requests.

## To Test

https://sandbox.uiowa.ddev.site/people-directory should match their view config and work correctly on-click. Pagers should also work for block/page re-save and after `cr`.
